src/lib/components/molecules/control-change/index.jsx

```diff
@@ -20,7 +20,7 @@ export const ControlChange = ({ previous, next, text, styles }) => {
             }}
           />
         ) : (
-          <CircleIcon styles={{ circle: `fill-color--${next}` }} />
+          <CircleIcon styles={{ circle: `fill-color--${styles.next}` }} />
```

Comment:
I hadn't really noticed this before, but shouldn't `style.next` be directly assigned to `circle`, i.e. `{ circle: styles.next }`?

This would be slightly unexpected to me, because the logic for those style overrides elsewhere is that you pass a full class name that you want to apply to an element to customise the component. It's not just any string.